client/Cargo.toml

```diff
@@ -9,6 +9,8 @@ tpm-simulator-tests = []
 [dependencies]
 tpm2-rs-base = { workspace = true }
 tpm2-rs-marshalable = { workspace = true }
+tpm2-rs-unionify = { workspace = true }
+tracing = { workspace = true }
```

Comment:
:(

Hard disagree.

If you needed this for testing, I would have added it to dev-dependencies.

Reply:
Oh, I thought you already merged this, but it's actually still in review - my bad.

Anyways, I don't think we need this, personally. I don't see what this really adds that we don't get well-enough with just `println` in test situations, but I'd be okay adding it as long as you only add it to `dev-dependencies`

Reply:
I have moved all texts into the assert line, so that we only get the print in case of a failed test 👍 🙂

Reply:
Thanks! I think this is the way forward with our tests (adding details to the assertions), so hopefully that's all we need. :)